sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -120,7 +120,7 @@ class CacheManager extends Logging {
   def uncacheQuery(
       query: Dataset[_],
       cascade: Boolean,
-      blocking: Boolean = true): Unit = writeLock {
+      blocking: Boolean = true): Unit = readLock {
```

Comment:
why do we need the `readLock` here?

Reply:
I was just converting the prior readLock, but it isn't needed.  I'll delete.